src/ipvs/ip_vs_laddr.c

```diff
@@ -525,6 +525,7 @@ static int get_msg_cb(struct dpvs_msg *msg)
 
     *laddrs = *laddr_conf;
     laddrs->nladdrs = naddr;
+    laddrs->cid = cid;
```

Comment:
the `laddrs->cid` is set by the user defined input param "conf" in `laddr_sockopt_get`. It's no need to set the param here.

Reply:
the laddrs->cid  set by the user defined input param "conf" is  cpu index , which is not equal to lcore id. 
eg: 
start dpvs with -l  1,2,3,4,5; the cpu index of  slave worker with lcore id 2  is  1.

Reply:
ok.  you are right!